pkg/tfbridge/schema.go

```diff
@@ -1150,7 +1175,8 @@ func makeConfig(v interface{}) interface{} {
 // MakeTerraformConfigFromInputs creates a new Terraform configuration object from a set of Terraform inputs.
 func MakeTerraformConfigFromInputs(p shim.Provider, inputs map[string]interface{}) shim.ResourceConfig {
 	raw := makeConfig(inputs).(map[string]interface{})
-	return p.NewResourceConfig(raw)
+	config := p.NewResourceConfig(raw)
+	return config
```

Comment:
I don't agree with this change.

Reply:
mb, I had some debugging here and accidentally left the change after.